server/tserver/src/main/java/org/apache/accumulo/tserver/compaction/CompactionStrategy.java

```diff
@@ -28,6 +28,12 @@
 /**
  * The interface for customizing major compactions.
  * <p>
+ * Important: Compaction configurations should use one compaction scheme, either the
```

Comment:
This is not quite correct.  The compaction strategy was a per table or per compaction configured item, the replacement for its functionality are the CompactionSelector and CompactionConfigurer plugins which can also be configured per table or per compaction.   An attempt to explain this is here[here](https://github.com/apache/accumulo/blob/e6cbdb4418d50e6c3c99e166a0390e483401fded/core/src/main/java/org/apache/accumulo/core/client/admin/CompactionStrategyConfig.java#L40C85-L40C85).

The CompactionPlanner is not a replacement for the CompactionStrategy.  The compaction planner is a tserver level plugin and it break files selected by the CompactionStrategy (if confugred) into one or more compaction jobs that are scheduled on thread pools.